test/sagemaker_tests/pytorch/inference/integration/__init__.py

```diff
@@ -39,32 +39,3 @@
 DEFAULT_TIMEOUT = 20
 
 RESOURCE_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "resources"))
-
```

Comment:
I checked around in the code and it appears safe to remove these - only found one usage for both NO_P2 and NO_P3

Reply:
Yeah, modified the wrong list of regions the first time around. These are not used anywhere in the code.